### ISSUE TITLE
Remove Mapbox base mapes

### ISSUE THREAD

**Issue by tpcolson** [tpcolson/npmap-species#176]
Remove mapbox base maps from code as we no longer are authorized to use mapbox.

**Comment by 3PIV** [comment]
```

<div id='search-options' class='bordered'>
--
  | <div id='search-options-left'>
  | <div id='options-label'>MAP CONTROLS</div>
  | <div id='options-background' tooltip='Change the background map'>
  | <div id='options-background-dropdown' class='species-dropdown' onkeypress='toggleBackgroundList();'
  | onclick='toggleBackgroundList();' data-intro="Change map background" data-position='left'>
  | Map Background
  | <div class='triangle'></div>
  | <ul id='options-background-dropdown-ul'>
  | <li onkeypress='updateBaseLayer(0);' onclick='updateBaseLayer(0);'>&#10004; Park Tiles</li>
  | <li onkeypress='updateBaseLayer(1);' onclick='updateBaseLayer(1);'>Mapbox Terrain</li>
  | <li onkeypress='updateBaseLayer(2);' onclick='updateBaseLayer(2);'>Esri Topo</li>
  | <li onkeypress='updateBaseLayer(3);' onclick='updateBaseLayer(3);'>Esri Imagery</li>
  | </ul>
  | </div>
  | </div>


```
The offending code.

**Comment by 3PIV** [comment]
This issue has been fixed by: 510f110 . The Mapbox tiles and bad Esri tiles were replaced by NPS slate and NPS satellite imagery.
